Bug batching for Scalability and Performance 4.16 (#77809)
openshift/openshift-docs: release_notes/ocp-4-16-release-notes.adoc

[id="ocp-4-16-scalability-and-performance-bug-fixes_{context}"]
==== Scalability and performance

> * Previously, the Machine Config Operator (MCO) on single-node {product-title} was rendered after the Performance Profile rendered, so the control plane and worker MachineConfigPools were not created at the right time. With this release, the PerformanceProfile renders correctly and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-22095[*OCPBUGS-22095*])

Reviewer: ```suggestion
* Previously, the Machine Config Operator (MCO) on single-node {product-title} was rendered after the Performance Profile rendered, so the control plane and worker machine config pools were not created at the right time. With this release, the Performance Profile renders correctly and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-22095[*OCPBUGS-22095*])
```

Reviewer: Perhaps it doesn't matter, but would it be more accurate:

```suggestion
* Previously, the Performance Profile on single-node {product-title} was rendered before the Machine Config Operator (MCO) was rendered, so the control plane and worker machine config pools were not created at the right time. With this release, the Performance Profile renders correctly and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-22095[*OCPBUGS-22095*])
```

> * Previously, the TuneD and `irqbalanced` daemons modified the Interrupt Request (IRQ) CPU affinity configuration, which created conflicts in the IRQ CPU affinity configuration and caused unexpected behaviour after a {sno} node restart. With this release, only the `irqbalanced` daemon determines IRQ CPU affinity configuration. (link:https://issues.redhat.com/browse/OCPBUGS-26400[*OCPBUGS-26400*])

Reviewer: ```suggestion
* Previously, the TuneD and `irqbalanced` daemons modified the Interrupt Request (IRQ) CPU affinity configuration, which created conflicts in the IRQ CPU affinity configuration and caused unexpected behavior after a {sno} node restart. With this release, only the `irqbalanced` daemon determines IRQ CPU affinity configuration. (link:https://issues.redhat.com/browse/OCPBUGS-26400[*OCPBUGS-26400*])
```
